### PR TITLE
TypeError: NoneType object is not iterable

### PR DESCRIPTION
**Pull request by ennozdd**
## Related Issues
fixes:

## Description
internalMapping is set to null in my case and validation fails with "TypeError: NoneType object is not iterable" 

        "dbot_classification_incident_type_all": {
            "dontMapEventToLabels": false,
            "internalMapping": null
        }

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/mapper.py [hunk 1]
```diff
@@ -98,6 +98,10 @@ def is_field_mapping_removed(self):
             for inc in old_incidents_types:
                 old_incident_fields = old_mapper[inc].get("internalMapping", {})
```
Comment (YuvHayun) [inline demisto_sdk/commands/common/hook_validations/mapper.py:99]:
```suggestion
                old_incident_fields = old_mapper[inc].get("internalMapping", {}) or {}
```

File: demisto_sdk/commands/common/hook_validations/mapper.py [hunk 2]
```diff
@@ -98,6 +98,10 @@ def is_field_mapping_removed(self):
...
                 current_incident_fields = current_mapper[inc].get("internalMapping", {})
```
Comment (YuvHayun) [inline demisto_sdk/commands/common/hook_validations/mapper.py:100]:
```suggestion
                current_incident_fields = current_mapper[inc].get("internalMapping", {}) or {}
```

File: demisto_sdk/commands/common/hook_validations/mapper.py [hunk 3]
```diff
@@ -98,6 +98,10 @@ def is_field_mapping_removed(self):
...
+                if old_incident_fields is None:
+                    old_incident_fields = {}
+                if current_incident_fields is None:
+                    current_incident_fields = {}
```
Comment (YuvHayun) [inline demisto_sdk/commands/common/hook_validations/mapper.py:104]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -15,6 +15,7 @@
 * Fixed an issue where **download** command failed when running on a beta integration.
 * Added the `DEMISTO_SDK_GRAPH_FORCE_CREATE` environment variable. Use it to force the SDK to recreate the graph, rather than update it.
 * Added support for code importing multi-level ApiModules to **lint**.
+* Fixed an issue where **validate** failed on null values in a classifier.
```
Comment (YuvHayun) [inline CHANGELOG.md:18]:
```suggestion
* Fixed an issue where **validate** failed when attempting to map null values in a classifier.
```

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hey @ennozdd.
Nice catch and thx for your contribution!
I made small adjustments, could you please add the changes and confirm this fixes your issue?
Let me know if it does and I'll approve the pr.
Feel free to contact me here / Via Slack if there are any questions.

**Comment by ennozdd** [comment]
@YuvHayun done

File: CHANGELOG.md [hunk 5]
```diff
@@ -26,6 +27,7 @@
 * Fixed an issue with **update-content-graph** where `--use-git` parameter was ignored when using `--imported-path` parameter.
 * Fixed an issue where **validate** failed on playbooks with valid inputs, since it did not collect the playbook inputs occurrences properly.
 
+
```
Comment (YuvHayun) [inline CHANGELOG.md:30]: redundant
```suggestion
```

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**